src/DropDownMenu/DropDownMenu.spec.js

```diff
@@ -31,6 +32,43 @@ describe('<DropDownMenu />', () => {
     });
   });
 
+  describe('prop: disabled', () => {
+    it('disables the touchRipple effect if disableTouchRipple is undefined and disabled={true}', () => {
+      const wrapper = mountWithContext(
```

Comment:
The reason I used mount is due to my testing strategy, as I had checked for the existence of a `TouchRipple` component. There definitely has to be a better strategy though